README.md

```diff
@@ -209,15 +256,26 @@ date**.
     you didn't uncheck `Run MSYS2 now` as instructed.  That shell will not work,
     so close it if you forgot to uncheck it in the installer.
 - From within the `MSYS2 MINGW64` shell enter the following commands to install
-  `gcc`, `git`, `go`, `unzip`, the light OpenCL SDK, and the `gominer` source code
+  `gcc`, `git`, `go`, `unzip`:
   - `pacman -S mingw-w64-x86_64-gcc mingw-w64-x86_64-tools mingw-w64-x86_64-go git unzip`
+  - `git clone https://github.com/decred/gominer`
+- Jump to the appropriate section for either
+  [OpenCL](#opencl-pre-requisites-on-windows), or [CUDA with NVIDIA](#cuda-with-nvidia) 
+  depending on which type of GPU you have
+
+##### OpenCL Pre-requisites on Windows
+
+The following is needed when performing an OpenCL build:
+
+- Still in the `MSYS2 MINGW64` shell enter the following commands to install 
+  the light OpenCL SDK:
   - `wget https://github.com/GPUOpen-LibrariesAndSDKs/OCL-SDK/files/1406216/lightOCLSDK.zip`
   - `unzip -d /c/appsdk lightOCLSDK.zip`
-  - `git clone https://github.com/decred/gominer`
 - **Close the `MSYS2 MINGW64` shell and relaunch it**
```

Comment:
I'm fairly certain you'll need this up in the prereqs section for both opencl and cuda (prior to the "Jump to the appropriate...").  It's really for the gcc and Go env variables.  Unzipping the opencl sdk doesn't need the shell closed and reopened.